src/smc-hub/client.coffee

```diff
@@ -1407,25 +1408,11 @@ class exports.Client extends EventEmitter
                     if mesg.subject?
                         subject  = mesg.subject
 
-                    if mesg.link2proj? # make sure invitees know where to go
-                        base_url = mesg.link2proj.split("/")
-                        base_url = "#{base_url[0]}//#{base_url[2]}"
-                        direct_link = "Open <a href='#{mesg.link2proj}'>the project '#{mesg.title}'</a>."
-                    else # fallback for outdated clients
-                        base_url = 'https://cocalc.com/'
-                        direct_link = ''
-
-                    email_body = (mesg.email ? '') + """
-                        <br/><br/>
-                        <b>To accept the invitation, please open
-                        <a href='#{base_url}'>#{base_url}</a>
-                        and sign in using your email address '#{locals.email_address}'.
-                        #{direct_link}</b><br/>
-                        """
-
-                    # The following is only for backwards compatibility with outdated webapp clients during the transition period
-                    if not mesg.title?
-                        subject = "Invitation to CoCalc for collaborating on a project"
+                    try
+                        email_body = makeEmailBody(subject, mesg.email,locals.email_address,  mesg.title, mesg.link2proj)
```

Comment:
It would be nice to use Javascript's standard camelCase variable/function names.  However, we do that basically nowhere in our entire codebase, and not even elsewhere in this same line (e.g., locals.email_address)... I feel like randomly introducing it will just be confusing/annoying, and make us have to think about stuff we don't really have time to think about...  We could discuss this tomorrow morning.